stage-prep/06-automount/01-run.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -e
+
+set -exu
+
+install -v -m 0644 files/media-automount@.service "${ROOTFS_DIR}/usr/lib/systemd/system/media-automount@.service"
```

Comment:
This has to run as a system service? Because of `sudo mount ...`? Is that correct?